ftl_installer/parser/Parser.py

```diff
@@ -18,6 +20,8 @@ def __init__(self):
         self.parser.add_argument('-q', '--quiet', action='store_true', dest='quiet',
                                  default=False, help='supress output')
 
+        self.parser.add_argument('-c', '--config', help='config file')
```

Comment:
We'll want to support a default config file location. Eventually this will need to be user customizable, but for now I think we can just add a default value to -c option here. I would suggest `~/.config/ftl_installer`